packages/documentation/docs/installation/installation.md

```diff
@@ -23,4 +24,11 @@ Follow the links below to the respective installation guides:
   <Card link="installation/javascript"><WebComponentsIcon class="Card_Icon" />Web Components</Card>
   <Card link="installation/vue"><VueIcon class="Card_Icon" />Vue&nbsp;<span style={{fontSize: '0.8rem'}}>(experimental)</span></Card>
   <Card link="installation/blazor"><BlazorIcon class="Card_Icon" />Blazor&nbsp;<span style={{fontSize: '0.8rem'}}>(experimental)</span></Card>
-</CardList>
+</CardList>  
+
+   
+**Experience the Industrial Experience style while coding**  
+We have designed a VS Code theme for our Siemens AG employees and business partners to experience the Industrial Experience style while coding. 
+Download it here:
+
+<Card link="https://code.siemens.com/siemens-ix/ix-for-vs-code/ix-theme-for-vs-code/-/releases"><VSCodeIcon class="Card_Icon" />VS Code theme</Card>
```

Comment:
This links to an internal repository. External users will not be authorised to view it. Maybe we could add a disclaimer?

Reply:
That was also my first idea and i used the class" siemens-brand-section" for the description text but i had the impression that this would catch too much attention compared to the other content on the page. What do you think? Any other ideas?

Reply:
maybe it is enough to add to the line above "Download it here (available within Siemens)"?

Reply:
Yes, i could add it. I would look like this: ![image](https://github.com/user-attachments/assets/963376f8-dab2-4ff6-b641-a17e17de1a89)
Should i commit it?

Reply:
Latest update: 
![image](https://github.com/user-attachments/assets/15568384-801b-4750-8fcc-2654d6a37c76)
I removed to VS Code icon because we are not allowed to use it in this context. I also updated the naming and included your proposal.